exonum-java-binding/core/src/main/proto/service.proto

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package exonum.binding;
+
+option java_package = "com.exonum.binding.common.messages";
+
+/**
+TODO: move this file to exonum-proto-sources-repo
+currently this action is blocked till native update to the latest revision
+*/
+
+// An object for the service configuration which passed to `service#initialize` method.
+// It brings a possibility to configure a service using any of provided formats.
+// The format provides the type of the configuration encoded as string value.
```

Comment:
Service configuration arguments in a text format.
The string must be in one of the pre-defined formats.